src/utils/bitConversion.js

```diff
@@ -55,13 +54,41 @@ export const convertBits16 = b8 =>
     ((b8 & 1) + 1);
 
 /**
- * @function doubleBitValue
+ * Extract out of a 16 bit value the averaged bit state.
+ * There are 3 valid states as described in the return value.
+ *
  * @param {number} b16 16-bit input of 8 bit-pairs
  * @param {number} n index of bit-pair
  * @returns {number}
  * - 0 (b00): invalid (undefined)
- * - 1 (b01): was all 0
- * - 2 (b10): was all 1
- * - 3 (b11): was both 0 and 1
+ * - 1 (b01): was always 0
+ * - 2 (b10): was always 1
+ * - 3 (b11): was sometimes 0 and sometimes 1
  */
-export const doubleBitValue = (b16, n) => 3 & (b16 >> (n + n));
+export const averagedBitState = (b16, n) => 3 & (b16 >> (n + n));
+
+export const always0 = 1;
+export const always1 = 2;
+export const sometimes0And1 = 3;
+
+const lineDateFor1 = 0.4;
+const lineDateFor0 = -lineDateFor1;
```

Comment:
I think these are typos, probably meant to be _lineDataFor..._.

Reply:
Thanks, fixed that in a4f8412